<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>searchPhoneNumber</class>
 <widget class="QDialog" name="searchPhoneNumber">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search Member</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
	font: 13pt &quot;Arial Rounded MT Bold&quot;;
background-color:rgba(1,25,53,200);
color:rgba(208,233,255);
border-color:rgba(1,25,53,90);
border-radius:3px;
}
QLabel
{
font: 11pt &quot;Arial&quot;;
color:rgb(1, 25, 53);
}
QLineEdit
{
border-color:rgba(255,255,255,200);
font: 11pt &quot;Arial&quot;;
border-color:rgba(1,25,53,50);
           border-radius:3px;
         background-color:rgba(255,255,255);
      color:rgba(1,25,53);
}
QRadioButton
{
font: 11pt &quot;Arial Rounded MT Bold&quot;;
color:rgb(1, 25, 53);
}
</string>
  </property>
  <widget class="QWidget" name="gridWidget" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>331</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="phoneNumber_label">
        <property name="text">
         <string>Phone Number:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="enterPhoneNum"/>
      </item>
      <item>
       <widget class="QPushButton" name="searchButt">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="discountLabel">
      <property name="text">
       <string>VIP Discount:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="pointLabel">
      <property name="text">
       <string>Total Pointï¼š</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="nameLabel">
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="displayDiscount">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="displayMemberName">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="displayMemberPoint">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QWidget" name="horizontalWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="radioButt_vip">
         <property name="text">
          <string>VIP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_superVip">
         <property name="text">
          <string>Super VIP</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="skipButt">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Skip</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButt">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>290</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>gridWidget</zorder>
  <zorder>skipButt</zorder>
  <zorder>okButt</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
